tests/cypress/e2e/features/ground_truth_jobs.js

```diff
@@ -240,7 +270,7 @@ context('Ground truth jobs', () => {
                 cy.get('.cvat-player-next-button').click();
             });
 
-            cy.checkFrameNum(groundTruthFrames[2]);
+            cy.checkFrameNum(groundTruthFrames[groundTruthFrames.length - 1]);
```

Comment:
We already have checking frame number above in `forEach`, is not it?

Reply:
yes, removed that